Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue1939.cs

```diff
@@ -7,11 +7,16 @@
 #if UITEST
 using Xamarin.UITest;
 using NUnit.Framework;
+using Xamarin.Forms.Core.UITests;
 #endif
 
 namespace Xamarin.Forms.Controls.Issues
 {
 	[Preserve(AllMembers = true)]
+#if UITEST
+	[Ignore("Ignoring this test as it fails sometimes on TestCloud")]
```

Comment:
I'm hoping this https://github.com/xamarin/Xamarin.Forms/pull/6562 fixes this test

It looks like the failure here is coming from the test that runs before it so I don't know if ignoring this test will fix